reason-react-native/src/apis/Style.bs.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+var Js_dict = require("bs-platform/lib/js/js_dict.js");
+
+function unsafeAddProp(style, property, value) {
```

Comment:
Did you forget to update the .bs.js file? It does not correspond to the .re file.

Reply:
No. Compilation fails & I didn't yet find why. Saying the .rei doesn't match the .re but I don't see the issue :/